examples/cluster.yaml

```diff
@@ -59,12 +59,22 @@ spec:
     keyVault:
       name: "example-cluster-vault"
     location: "westeurope"
+    masters:
+    - vmSize: "Standard_A1"
+      dataDiskSizeGB: 50
+      adminUsername: "core"
+      adminSSHKeyData: "ssh-rsa AAAAB3NzaC1y user@location"
+      osImage:
+        publisher: "CoreOs"
+        offer: "CoreOS"
+        sku: "Stable"
+        version: "1465.7.0"
     storage:
       accountType: "Standard_LRS"
     virtualNetwork:
       cidr: "10.0.0.0/16"
       masterSubnetCidr: "10.0.1.0/24"
-      workerSubnetCidr: "10.0.2.0/24"      
+      workerSubnetCidr: "10.0.2.0/24"
```

Comment:
I tried to create a cluster according to the local guide and failed with:

```
{"caller":"github.com/giantswarm/azure-operator/vendor/github.com/giantswarm/operatorkit/framework/framework.go:102","error":"[{/Users/pawel/go/src/github.com/giantswarm/azure-operator/vendor/github.com/giantswarm/operatorkit/framework/framework.go:222: }
 {/Users/pawel/go/src/github.com/giantswarm/azure-operator/service/resource/deployment/resource.go:242: } {resources.DeploymentsClient#CreateOrUpdate: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code=\"InvalidTemplate\" Message=\"Deployment template validation failed: 'The template parameters 'masterCloudConfigData, mastersCustomConfig' in the parameters file are not valid; they are not present in the original template and can therefore not be provided at deployment time. The only supported parameters for this template are 'clusterID, storageAccountType, virtualNetworkCidr, masterSubnetCidr, workerSubnetCidr, apiLoadBalancerCidr, etcdLoadBalancerCidr, ingressLoadBalancerCidr, kubernetesAPISecurePort, etcdPort, kubernetesIngressSecurePort, kubernetesIngressInsecurePort, keyVaultName, keyVaultSecretsObject, templatesBaseURI, certificatesTemplateFile, clusterSetupTemplateFile, securityGroupsSetupTemplateFile, virtualNetworkSetupTemplateFile'. Please see https://aka.ms/arm-deploy/#parameter-file for usage details.'.\"}]","event":"create","time":"17-10-05 14:28:30.563"}
```

Reply:
Make sure to `export AZURE_TEMPLATE_URI_VERSION="control_plane"`